src/API/Management/Users.php

```diff
@@ -1,88 +1,265 @@
 <?php
-
+/**
+ * Users endpoints for the Management API.
+ *
+ * @package Auth0\SDK\API\Management
+ */
 namespace Auth0\SDK\API\Management;
 
-use Auth0\SDK\API\Header\ContentType;
-
-class Users extends GenericResource {
-
-    public function get($user_id) {
-
-        return $this->apiClient->get()
-            ->users($user_id)
+/**
+ * Class Users.
+ * Handles requests to the Users endpoint of the v2 Management API.
+ *
+ * @package Auth0\SDK\API\Management
+ */
+class Users extends GenericResource
+{
+    /**
+     * Get a single User by ID.
+     * Requires scopes: read:users, read:user_idp_tokens.
```

Comment:
Suggestion -> Enclose values in quotes.  (applies for all docblocks)

IMHO required -> explain _when_ the `read:user_idp_tokens` scope would be required. It's not that to call this endpoint you must provide it, only if you want to obtain the access_tokens for each logged in identity provider.